markdown/dev/tutorials/pattern-design/adding-measurements/en.md

```diff
@@ -33,9 +33,9 @@ Now everybody knows your pattern requires the `head` measurement.
 
 This change will also get picked up by the development environment, and you'll now see this screen:
 
-![This screen tells you you are missing some required measurements](./required-measurements.png)
+![This screen tells you are missing some required measurements](./required-measurements.png)
```

Comment:
Yeah fair point, would suggest "This screen tells you `that` you are missing some required measurements"

A pull request if you're happy with this is in https://github.com/freesewing/freesewing/pull/3126

Otherwise no worries if you want to leave it as is